var/spack/repos/builtin/packages/hcol/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hcol(Package):
+    """This is the SPIRAL package for the Hybrid Control Operator Language
+    (HCOL)."""
+
+    homepage = "https://https://spiral.net"
+    url      = "https://github.com/spiral-software/spiral-package-hcol/archive/1.0.0.zip"
+
+    maintainers = ['spiralgen']
+    extends('spiral')
+
+    version('1.0.0', sha256='9a95574e2b061d03d264c32dbf733e893017d1644b6486c7a8a55a3b24783f58')
+
+    # HCOL package is a set of files to be installed in the
+    # "namespaces/packages" folder (subfolder hcol for this package).
+    # No other operations are required to build.
```

Comment:
This might be problematic. Spack's philosophy on package installation is to install each package into its own prefix. Without that, uninstalling the package won't actually uninstall it.

You could either add this as a resource to the `spiral` package, or you could implement activation/deactivation so that `spack install hcol && spack activate hcol` would install `hcol` to its own prefix, then symlink it to the `spiral` installation prefix. See the `python` package for an example where this is done. I don't personally have experience messing with this unfortunately.

Reply:
Changed so that hcol is installed in its own prefix.  A symlink is used from spiral to link to this.